### ISSUE TITLE
Validation fails when it should pass - "Audio element content has text alternative"

### ISSUE THREAD

**Issue by gavinmitchell-qub** [opena11y/ainspector#8]
Hi,

I'm using the Accessibility Inspector tool extension in Firefox (version 4.1.4) to validate some sites.  I've come across an issue that I think should pass validation, but doesn't.

Audio/Video->
Audio-only (Prerecorded)->
Violation: Provide a text transcript for audio element content.

I think this rule is defined here
https://www.w3.org/TR/WCAG22/#audio-only-and-video-only-prerecorded
https://www.w3.org/WAI/WCAG22/Techniques/general/G158

The link above has some test cases which do not seem to pass in the tool - 
https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/e7aa44/85c98d1402dbc9c68ace2fbf5f063d145b8e5bd7.html
https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/e7aa44/dedfb667190bd564527247550565cdea8ccefd3f.html

These both fail even though the pages have text transcripts

<img width="961" height="935" alt="Image" src="https://github.com/user-attachments/assets/57b4ade1-8627-45ad-9688-ba18e52be701" />

Many thanks for this great tool !

Gavin
